authors/apps/articles/tests/test_model.py

```diff
@@ -0,0 +1,23 @@
+from ..models import Article
+from .base_class import ArticlesBaseTest
+from ...profiles.models import Profile
+
+
+class TestArticleModel(ArticlesBaseTest):
+    """This Test class tests the article models __str__ method
+    """
+
+    def test_article_representation(self):
+        article = self.create_article()
+        self.assertEqual(str(article), 'This is the title')
+
+    def create_article(self):
+        # login
+        self.register_and_login_user()
+        article = Article.objects.create(
+            title='This is the title',
+            description='This is the description',
+            body='This is the body',
+            author=Profile.objects.first(),
+            tag_list=['chill', 'chills'])
```

Comment:
Right now the Articles should not include tags because that was not defined in your feature on the PT board